packages/core/logging/core-logging-common-internal/src/browser_config.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { LogLevelId } from '@kbn/logging';
+
+/**
+ * @internal
+ */
+export interface BrowserLoggingConfig {
+  root: BrowserRootLoggerConfig;
+}
+
+/**
+ * @internal
+ */
+export interface BrowserRootLoggerConfig {
+  level: LogLevelId;
+}
```

Comment:
Base types for the browser-side logging config. It's simplistic right now, but more fields should appear when we go further on https://github.com/elastic/kibana/issues/144276